### PR TITLE
ISSUE-71 Feature/exception-handling 

### PR DESCRIPTION
**Pull request by kaansonmezoz**

### CODE REVIEW

**Comment by hannahkamundson** [comment]
@kaansonmezoz sent you an email with discord link!

**Comment by kaansonmezoz** [comment]
I have forgotten the unit tests when I open pull request. It will be ready for tomorrow

File: Backend/src/main/java/idealab/api/controller/PrintJobController.java [hunk 1]
```diff
@@ -1,30 +1,24 @@
...
-import org.springframework.web.bind.annotation.RestController;
-
 import org.springframework.web.bind.annotation.*;
 
 @RestController
 @RequestMapping("/api/printjobs")
 public class PrintJobController {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(PrintJobController.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(PrintJobController.class); // TODO: loglama islemini baska bir yerde yapmak lazim !
```
Comment (Will-Estes) [inline Backend/src/main/java/idealab/api/controller/PrintJobController.java]: Please put comments in English.

File: Backend/src/main/java/idealab/api/dto/response/ExceptionResponse.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
+package idealab.api.dto.response;
+
+import java.time.LocalDateTime;
+
+public class ExceptionResponse { // Bunun icin bir builder pattern olusturabiliriz aslinda daha hos durur.
```
Comment (Will-Estes) [inline Backend/src/main/java/idealab/api/dto/response/ExceptionResponse.java]: Please make comments in English

File: Backend/src/main/resources/application.properties [hunk 3]
```diff
@@ -6,10 +6,14 @@ spring.jpa.database=POSTGRESQL
 spring.datasource.platform=postgres
 spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
 spring.datasource.username=postgres
-spring.datasource.password=docker
```
Comment (Will-Estes) [inline Backend/src/main/resources/application.properties]: Why update this? Please change back if not necessary.

**Pull request merged**